### DISCUSSION TITLE
Is there a reason why Sandboxie Control is shipped with Sandboxie Plus?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#2763] [General]
Up to Sandboxie Plus 1.8.1 (and beyond, supposedly) Sandboxie Control, SbieCtrl.exe and it's DLL, are included in the installer and being installed alongside the modern UI.

Is there a reason why it gets bundled and installed? Does it serve a purpose, even when the new UI (sandman.exe) is used?

**Comment by DavidXanatos** [comment]
No it does not serve a purpose SbieCtrl.exe was initially put there as the plus UI was in the early days incomplete and no one knew how to use it, so it seamed expedient to provide the old UI als an alternative when a user is not sure how to do what that want in the ini directly.
Nowadays it perhaps really should get removed from the release.

**Reply by bastik-1001** [reply]
I assume that anyone that does not like the new UI uses the classic version anyway. It was not obvious to me, that Sandboxie Control, was just being shipped for transition, even though I guessed so.

**Reply by stdedos** [reply]
I kinda liked to have the SbieCtrl.exe install alongside the SandmanUI, since the latter was incomplete.

However, Sandman has grown up substantially, I guess no one would really need that thing.
Or idk, start a poll for that until the final removal decision 😝
